fastlane/Fastfile

```diff
@@ -15,6 +15,7 @@ derived_data_path = 'derived_data'
 source_packages_path = 'spm_cache'
 buildcache_xcargs = 'CC=clang CPLUSPLUS=clang++ LD=clang LDPLUSPLUS=clang++'
 is_localhost = !is_ci
+project_package_resolved = './../StreamChatSwiftUI.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved'
```

Comment:
`./../` might be confusing/not available in some cases. I think better to use the root folder path and check it out where required via `Dir.chdir('..')` as we usually do.

```suggestion
project_package_resolved = "#{xcode_project}/project.xcworkspace/xcshareddata/swiftpm/Package.resolved"
```